osquery/tables/system/posix/authorized_keys.cpp

```diff
@@ -48,13 +68,78 @@ void genSSHkeysForUser(const std::string& uid,
       logger.log(google::GLOG_WARNING, s.getMessage());
       continue;
     }
+
     // Protocol 1 public key consist of: options, bits, exponent, modulus,
     // comment; Protocol 2 public key consist of: options, keytype,
     // base64-encoded key, comment.
     for (const auto& line : split(keys_content, "\n")) {
       if (!line.empty() && line[0] != '#') {
```

Comment:
I know this is prior code, but this would probably be a little cleaner as an early return, and another nest layer

Reply:
If so, the log severity is changed to error when that error is unrecoverable.